apache/doris-operator
# [Refactor]Clear resource refactor

### What problem does this PR solve?
1. refactor the clear resource logic.
2. fixed the service label error.
3. use statefulset replicas to replace compute group's replica as the baseline for clear pvc.
4. clear the unused codes for tiding.

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
        1. deploy multi compute groups.
        2. reduce the first compute group replicas and delete second comute group.
        5. verify the  svc , sts , PVC are equal to the final state.

- Behavior changed:

- Does this need documentation?

### Check List (For Reviewer who merge this PR)
